Source/BSN.IpTables.Cli/configuration.yaml

```diff
@@ -7,14 +7,16 @@ subject-prefix: $(service-name)
 inlining-threshold: 100
 
 tag: v1
-input-file: ../BSN.IpTables.Api/swagger.json
+input-file: ./swagger.json
```

Comment:
Are you sure?

I think we have to use swagger.json inside of `BSN.IpTables.Api`

Reply:
Yes, it was changed for test purposes. I reverted that.